ps2-decompile/attach.c

```diff
@@ -3,188 +3,200 @@
 #include "callbacks.h"
 #include "shared.h"
 
+static const int USB_ID = 0x110;
```

Comment:
This should be put here with the leading 0: `0x0110`.

```suggestion
static const int USB_ID = 0x0110;
```